app/models/like.rb

```diff
@@ -0,0 +1,19 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  before_create :set_created_at
+  before_save :set_updated_at
+
+  def set_created_at
+    self.created_at ||= Time.current
+  end
+
+  def set_updated_at
+    self.updated_at = Time.current
+  end
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ Required ] I think it would be nice if you ensure that all custom methods in your models are also tested. To comply with the project requirements please add a unit test for the `update_likes_counter` method in the `Like` model.

## Example code suggestion for update_likes_counter_spec
```ruby
 require 'rails_helper'

RSpec.describe Like, type: :model do
  before(:all) do
    @user = User.create(
      name: 'Anne',
      photo: 'https://anne.com/ann.jpg',
      bio: 'This Annes bio',
      posts_counter: 0
    )
    @post = Post.create(
      title: 'A new dawn',
      text: 'Software Developer',
      author_id: @user.id,
      comments_counter: 0,
      likes_counter: 0
    )
  end

  it '@posts like_counter should increase by one' do
    Like.create(author_id: @user.id, post_id: @post.id)
    Like.create(author_id: @user.id, post_id: @post.id)
    post = Post.find(@post.id)
    expect(post.update_likes_counter).to eq(2)
  end
end
```

## Project requirements
![Screenshot from 2023-11-18 13-10-59](https://github.com/paulshaun-kdtk/blog-posts/assets/16392046/3884496b-33ca-40bd-b132-dbf684fa13c0)